### PR TITLE
[JS] chore: update setup for chatModeration and mathBot

### PR DESCRIPTION
**Pull request by lilyydu**
## Linked issues

closes: #967 and #968

## Details

1. Added in sample.env with associated keys.
2. Removed the AI-related variables from the local YMLs.
3. Removed duplicate key names in logic
4. Updated docs

## Attestation Checklist

- [x] My code follows the style guidelines of this project

- I have checked for/fixed spelling, linting, and other errors
- I have commented my code for clarity
- I have made corresponding changes to the documentation (we use [TypeDoc](https://typedoc.org/) to document our code)
- My changes generate no new warnings
- I have added tests that validates my changes, and provides sufficient test coverage. I have tested with:
  - Local testing
  - E2E testing in Teams
- New and existing unit tests pass locally with my changes

### CODE REVIEW

File: js/samples/05.chatModeration/README.md [hunk 1]
```diff
@@ -40,12 +40,17 @@ This sample shows how to incorporate Azure Open AI Content Safety control into a
...
     ```bash
     cd teams-ai/js/samples/05.chatModeration/
-    yarn start
     ```
 
+4. Duplicate the `sample.env` in the `teams-ai/js/samples/05.chatModeration` folder. Rename the file to `.env`. 
+
+5. If you are using OpenAI then only keep the `OPENAI_KEY` and add in your key. Otherwise if you are using AzureOpenAI then only keep the `AZURE_OPENAI_KEY`, `AZURE_OPENAI_ENDPOINT`, `AZURE_MODERATOR_KEY` & `AZURE_MODERATOR_ENDPOINT` variables and fill them in appropriately.
+
+6. Update `config.json` and `bot.ts` with your model deployment number.
```
Comment (singhk97) [inline js/samples/05.chatModeration/README.md:49]: number or name?

File: js/samples/05.chatModeration/teamsapp.local.yml [hunk 2]
```diff
@@ -69,23 +69,6 @@ provision:
 # Defines what the `deploy` lifecycle step does with Teams Toolkit.
 # Runs after `provision` during Start Debugging (F5) or run manually using `teamsfx deploy --env local`.
 deploy:
-  # Ensures that the project dependencies are installed.
-  # - uses: cli/runNpmCommand
-    # with:
-    #   args: install --no-audit
-  - uses: script
-    with:
-      run: yarn
```
Comment (blackchoey) [inline js/samples/05.chatModeration/teamsapp.local.yml:78]: Nice to have: we may still need this yarn command to install dependencies for people who only want to try the sample. So developers can navigate to this folder, provide the openai key and press F5 to try the sample directly.

I see there is step in readme to run yarn under the `js` folder, which installs dependencies for all the sub folders. I'm not sure whether we want to enable another way to try the sample without installing dependencies for all the js projects, as the sample is quite standalone.

Comment (lilyydu) [reply js/samples/05.chatModeration/teamsapp.local.yml:78]: Yup, great point Chaoyi. Kavin and I have been discussing this and right now, our standard is that we want the samples to consume the local package so the install has to happen in the root js folder. However, we may explore allowing the build in the sample folders. There are tradeoffs to both, especially with dependencies/blockers that we'll need to consider.

File: js/samples/05.chatModeration/README.md (hunk 1 above)
Comment (lilyydu) [reply js/samples/05.chatModeration/README.md:49]: nice catch, I meant name!

**Review by singhk97 — DISMISSED** [review]

**Review by corinagum — APPROVED** [review]

**Pull request merged**